mkdocs.yml

```diff
@@ -35,7 +38,6 @@ theme:
     accent: 'deep purple'
   features:
     - navigation.tabs
-    - navigation.sections
```

Comment:
To display the left side menu with a drop-down, I removed this line. It is the only way I found to do what I wanted.
Open to any other suggestions.